extensions/wikia/Search/classes/Services/AbstractSearchService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Wikia\Search\Services;
+
+
+abstract class AbstractSearchService {
+
+	public function query( $phrase ) {
+		$select = $this->prepareQuery( $phrase );
+
+		$response = $this->select( $select );
+		$result = $this->consumeResponse( $response );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)